doc/en/components/grids/_shared/column-types.md

```diff
@@ -524,7 +524,7 @@ public editCellTemplate = (ctx: IgcCellTemplateContext) => {
 ```
 
 ```tsx
-function editCellTemplate(ctx: IgrCellTemplateContext) {
+function editCellTemplate(ctx: { dataContext: IgrCellTemplateContext }) {
```

Comment:
Add tree grid tag to the component start/end tags on line 389.

```
<!-- ComponentStart:  Grid, TreeGrid -->
Upon editing of cell's value the *currency symbol* will be visible as suffix or prefix. More about that could be found in the official [Cell editing topic](cell-editing.md#{PlatformLower}-grid-cell-editing-and-edit-templates-example).
<!-- ComponentEnd:  Grid, TreeGrid  -->
```

Reply:
This configuration throws an error as the link refers to the grid component
_cell-editing.md#{PlatformLower}-**grid**-cell-editing-and-edit-templates-example_

and the expected link for the tree grid will be
_cell-editing.md#{PlatformLower}-**tree-grid**-cell-editing-and-edit-templates-example_

TBH, I don't see a reason for this link pointing to the example, so I can configure it to point to the Cell Editing topic instead, however, it does not seem that this topic and the corresponding samples are enabled for the React tree grid so I assume this is the reason for the initial `ComponentStart/End Grid` tag.

Reply:
@RivaIvanova 

Sorry for the confusion. My intention was the link to direct to the general cell edit topic, not specifically to the sample. The documentation should indeed provide a comprehensive reference for understanding cell editing functionality.

It appears that while the [cell editing topic](https://github.com/IgniteUI/igniteui-xplat-docs/pull/1063) has been added, it hasn't been updated to show in the documentation with the `toc.json` file. As for the Cell Editing React samples for the tree grid, I'm unsure of their current development status. We can skip addressing that for now and revisit it at a later stage when we have more information.